### DISCUSSION TITLE
ইন্ডিকেটর শেপের শেডিং আপগ্রেড

### DISCUSSION THREAD

**Discussion by CoderBDK** [jummania/Jummania-Slider#14] [General]
হটাৎ একটা জিনিস খেয়াল করে দেখলাম যে, ইন্ডিকেটর শেপের কালার যেন কিরকম লাগতেছে, স্মুথনেস খুবই কম। তাই ভাবলাম। এটাকে একটু শেডিং করা যেতে পারে। তাহলে ইন্ডিকেটর শেপ দেখতে অনেক সুন্দর হবে আশা করি। এখন আপনাদের মতামত কি?

**Comment by jummania** [comment]
হ্যা, আপনি সঠিক। আমি শীঘ্রই নতুন আপডেটে এইসব দিকগুলো আপগ্রেড করবো।

**Reply by CoderBDK** [reply]
বিষয়টি বিবেচনা করার জন্য আপনাকে  অসংখ্য ধন্যবাদ।

**Reply by jummania** [reply]
বিষয়টি আমাদের নিকট তুলে ধরার জন্যে আপনাকেও অসংখ্য ধন্যবাদ
